### DISCUSSION TITLE
html-sanitizer of tgalopin is abandoned.

### DISCUSSION THREAD

**Discussion by abkrim** [filamentphp/filament#5890] [General]
Hi.
There's a message os last time and I see is a package used by filament, called [tgalopin/html-sanitizer](https://github.com/tgalopin/html-sanitizer)

This package is abandoned

> This library is deprecated as it was merged into Symfony as the HtmlSanitizer component in Symfony 6.1: https://symfony.com/doc/current/html_sanitizer.html

The same for [league/uri-parser](https://github.com/thephpleague/uri-parser)

> THIS PACKAGE IS ON MAINTENANCE MODE SINCE 2019-10-18, FOR ANY NEW PROJECT PLEASE USE [league/uri v6+](https://github.com/thephpleague/uri)

Best regards.

**Comment by danharrin** [comment]
We don't use `[league/uri-parser](https://github.com/thephpleague/uri-parser)` ourselves, whichever dependency uses that needs to upgrade.

We can't replace [tgalopin/html-sanitizer](https://github.com/tgalopin/html-sanitizer) with Symfony's component because it only supports PHP 8.1, and we still support PHP 8.0.
